go.mod

```
@@ -0,0 +1,13 @@
+module github.com/couchbase/go-blip
+
+go 1.18
```

Will this cause issues if `go-blip` is imported in packages that have `go 1.17` like SG?

---

Discussed offline - won't cause issues like build failures, but might be worth locking to 1.17 just to ensure that we don't accidentally use any 1.18 features before sync_gateway is upgraded to 1.18.
```suggestion
go 1.17
```

---

```suggestion
go 1.17
```